changelog/README.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 17th January 2023
+* Deprecated AssigneeData in [Get all bills](../operations/bills.md#get-all-bills) response
+* Extended [Bill customer data](../operations/bills.md#bill-customer-data) with `CustomerID`, `LegalIdentifiers`, `BillingCode`, `LastName`, `FirstName`, `SecondLastName`, `TitlePrefix` and with [Address](../operations/bills.md#address) fields.
```

Comment:
I might also add:
* Deprecated `ItalianFiscalData` in [Bill customer data](../operations/bills.md#bill-customer-data).